exporters/prometheus/src/exporter_utils.cc

```diff
@@ -114,8 +114,8 @@ std::vector<prometheus_client::MetricFamily> PrometheusExporterUtils::TranslateT
                 "invalid SumPointData type");
           }
         }
+        output.emplace_back(metric_family);
```

Comment:
Does the prometheus C++ library merge metric families?  We definitely don't want one metric family for each data point.

Reply:
thanks, creating new family only if it doesn't exist aleady.